src/mixins/filing-template-mixin.ts

```diff
@@ -468,7 +466,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       certifiedBy: filing.header.certifiedBy
     })
 
-    // NB: do not restore Folio Number - it was already set from business snapshot
```

Comment:
No, it wasn't. We load but we don't parse the business snapshot when we load a draft alteration.